security/libu2f-host/Portfile

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                libu2f-host
+version             1.1.3
+categories          security
+platforms           darwin
+maintainers         nomaintainer
+license             BSD
+
+description         Yubico Universal 2nd Factor (U2F) Host C Library
+
+long_description    Libu2f-host provides a C library and command-line tool \
+                    that implements the host-side of the U2F protocol. There \
+                    are APIs to talk to a U2F device and perform the U2F Register \
+                    and U2F Authenticate operations.
+
+homepage            https://developers.yubico.com/${name}/
+master_sites        https://developers.yubico.com/${name}/Releases/
+
+extract.suffix      .tar.xz
```

Comment:
This is usually done with
```tcl
use_xz                  yes
```
but I don't have any explanation or any reason against your approach. I'm not sure if you should listen to me in this case.